src/amber/cli/commands/encrypt.cr

```diff
@@ -19,6 +19,10 @@ module Amber::CLI
         encrypted_file = "config/environments/.#{env}.enc"
         unencrypted_file = "config/environments/#{env}.yml"
 
+        unless File.exists?(unencrypted_file) || File.exists?(encrypted_file)
+          raise Exceptions::Environment.new("./config/environments/", env)
```

Comment:
Exit?

Reply:
Also covered by existing rescue block. This is necessary for consistent messages. 

Again exiting isn't supposed to replace exceptions just handle them in a prettier way for the CLI.